### PR TITLE
Feature/lpad rpad

### PR DESCRIPTION
**Pull request by theodesp**

### CODE REVIEW

File: sql/expression/function/rpad_lpad.go [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+)
+
+type padType int
+
+const (
+	leftPadType  = iota
+	rightPadType
+)
+
+// MakePadder returns a Pad creator functions with a specific padType.
```
Comment (mcarmonaa) [inline sql/expression/function/rpad_lpad.go]: s/functions/function

File: sql/expression/function/rpad_lpad.go [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+func MakePadder(pType padType) func(e ...sql.Expression) (sql.Expression, error) {
+	return func(e ...sql.Expression) (sql.Expression, error) {
+		return NewPad(pType, e...)
+	}
+}
+
+// NewLogBase creates a new LogBase expression.
```
Comment (mcarmonaa) [inline sql/expression/function/rpad_lpad.go]: This comment doesn't match this function

File: sql/expression/function/rpad_lpad.go [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+		return nil, sql.ErrInvalidArgumentNumber.New("3", argLen)
+	}
+
+	return &Pad{args[0], args[1], args[2], pType}, nil
+}
+
+// Pad is a function that pads a string with another string.
+type Pad struct {
+	str     sql.Expression
+	len     sql.Expression
```
Comment (mcarmonaa) [inline sql/expression/function/rpad_lpad.go]: Since `len` is a builtin function in Go we could use another name, `length` maybe?

**Comment by theodesp** [comment]
PR updated

File: sql/expression/function/registry.go [hunk 4]
```diff
@@ -49,8 +49,10 @@ var Defaults = sql.Functions{
 	"json_extract":  sql.FunctionN(NewJSONExtract),
 	"connection_id": sql.Function0(NewConnectionID),
 	"soundex":       sql.Function1(NewSoundex),
-	"ln":            sql.Function1(LogBaseMaker(float64(math.E))),
-	"log2":          sql.Function1(LogBaseMaker(float64(2))),
-	"log10":         sql.Function1(LogBaseMaker(float64(10))),
-	"log":           sql.FunctionN(NewLog),
+	"ln": 			 sql.Function1(NewLogBaseFunc(float64(math.E))),
```
[the remark this answers is not in the document]
Comment (theodesp) [reply sql/expression/function/registry.go]: Sorry I forgot to show my whitespaces in my editor

**Pull request merged**